server/middleware/routes-acl/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function(options) {
+  options = options || {};
+  return function checkRouteAcls(req, res, next) {
+    // determine which route is being requested
+    // loop through routes-acl rules to check for matches
+    // if match
+    //    check the req user id and app id for perms
+    //    if allowed do nothing (let request through)
+    //    if not allowed
+    //      return 403
+    // if no match
+    //    do nothing (let request through)
```

Comment:
Sure. Ultimately, we should be able to just call the ACL engine that understands all the rules.